README.md

```diff
@@ -1 +1,237 @@
+  const info = await client.createAccount()
+  console.log(info) // will show account information
+})();
+~~~
+
+The `createAccount` method will try to fetch an existing account first and, if
+it does not exist, it will register it against the blockchain.
+
+You can safely use `createAccount` to fetch any account information, but you can
+also decide to just fetch it, without falling back to an account registration:
+
+~~~ts
+(async () => {
+  const info = await client.fetchAccountInfo()
+  console.log(info) // shows info (only if account exists, otherwise throws error)
+})();
+~~~
+
+### Vocdoni tokens; faucet & balance
+
+Accounts require Vocdoni tokens in order to be able to register against our
+blokchain. The process above will automatically fetch some tokens from a faucet
+**under development**. For production environments, you should contact us (Vocdoni)
+for a byte64 string faucet, and specify it when creating your account:
+
+~~~ts
+(async () => {
+  const info = await client.createAccount({
+    faucetPayload: "<b64string>"
+  })
+})();
+~~~
+
+Accounts also require Vocdoni tokens in order to create new processes.
+
+You can check the balance thanks to the previous methods (`createAccount` and/or
+`fetchAccountInfo`) and, under development, you can request new tokens thanks
+to the included faucet:
+
+~~~ts
+(async () => {
+  const info = await client.createAccount()
+  if (info.balance === 0) {
+    await client.collectFaucetTokens()
+  }
+})();
+~~~
+
+> Note: the `collectFaucetTokens` method only works under development environment
+> and for accounts with not enough tokens to create new processes.
+
+### Creating a voting process
+
+After you successfully registered your account against Vocdoni's blockchain, you
+can start creating processes.
+
+A process will require a census of people who will vote it. Let's start creating
+it:
+
+~~~ts
+const census = new PlainCensus()
+census.add(address)
+// "address" is a valid ethereum-alike address
+~~~
+
+When creating the process, we recommend you adding yourself to the census:
+
+~~~ts
+const census = new PlainCensus()
+// signers return addresses as promises, hence the usage of async here
+(async () => {
+  census.add(await signer.getAddress())
+})();
+~~~
+
+After you got all the addresses for your census, you may as well create the
+process instance:
+
+~~~ts
+const election = new Election({
+  title: 'Election title',
+  description: 'Election description',
+  // a header image for your process (this is for example purposes; avoid using random sources)
+  header: 'https://source.unsplash.com/random/2048x600',
+  startDate: new Date('2023-01-01 00:00:00'),
+  endDate: new Date('2023-01-23 23:23:23'),
+  census,
+})
+~~~
+
+Of course you will need some questions in this voting process, how would people
+vote otherwise?
+
+~~~ts
+election.addQuestion('Ain\'t this process awesome?', [
+  {
+    title: 'Yes',
+    value: 0,
+  },
+  {
+    title: 'No',
+    value: 1
+  }
+])
```

Comment:
I would attach another question, so we can show how to do it and it can be useful for showing how to vote

```ts
  election.addQuestion('This is a title', 'This is a description', [
    {
      title: 'Option 1',
      value: 0,
    },
    {
      title: 'Option 2',
      value: 1,
    },
  ])
  .addQuestion('This is a title', 'This is a description', [
    {
      title: 'Option 1',
      value: 0,
    },
    {
      title: 'Option 2',
      value: 1,
    },
  ]);
  ``